QCards/AppDelegate.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 
 @UIApplicationMain
-class AppDelegate: UIResponder, UIApplicationDelegate {
+class AppDelegate : UIResponder, UIApplicationDelegate {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/andreaslydemann/QCards/blob/3e62a0741f64e5d509d64d51f3765817c07b1c02/QCards/AppDelegate.swift#L12"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>AppDelegate.swift:12</code></td>
    </tr>
  </tbody>
</table>